charts/eks/values.yaml

```
@@ -196,14 +196,18 @@ syncer:
   serviceAnnotations: {}
   # Storage settings for the vcluster
   storage:
-   # If this is disabled, vcluster will use an emptyDir instead
-   # of a PersistentVolumeClaim
-   persistence: true
-   # Size of the persistent volume claim
-   size: 5Gi
-   # Optional StorageClass used for the pvc
-   # if empty default StorageClass defined in your host cluster will be used
-   #className:
+    # If this is disabled, vcluster will use an emptyDir instead
+    # of a PersistentVolumeClaim
+    persistence: true
+    # Size of the persistent volume claim
+    size: 5Gi
+    # Optional StorageClass used for the pvc
+    # if empty default StorageClass defined in your host cluster will be used
+    #className:
+    # Optional override for the "binaries" volume, in case the default emptyDir doesn't work for you
+    #binariesVolume:
+    # - name: binaries
+    #   emptyDir: {}
```

Not sure if I'm a fan of this. Effectively, there will only be one binaries volume, which must be called binaries.

I'd rather rewrite this to something like below, and embed it using a `toYaml` below the `- name: binaries` lines in the syncer.yaml.

```suggestion
    binariesVolume:
      emptyDir: {}
```